paper-badge.html

```diff
@@ -79,7 +87,14 @@
       }
     </style>
 
-    <div id="badge">{{label}}</div>
+    <div id="badge">
```

Comment:
We tend to discourage dom-ifs for simple cases like these, because they're actually a bit expensive. I think in this case `hidden` might be cheaper, or you might have to do some dom manipulations (manually create the `iron-icon` if the attribute changes).